.bazelci/presubmit.yml

```diff
@@ -35,4 +35,5 @@ tasks:
     - .bazelci/update_workspace_to_deps_heads.sh
     <<: *common
 
-buildifier: latest
+buildifier:
+  version: 4.0.1
```

Comment:
I pinned this because the surprise with native-py warning disappearing suggests that we shouldn't rely on consistent behavior when depending on floating "latest"

Reply:
im going to leave this floating since i dont think we'll be on top of updating this enough